### DISCUSSION TITLE
Customize the theme and styles of remirror

### DISCUSSION THREAD

**Discussion by devth** [remirror/remirror#1485] [Q&A]
I'm not finding a lot of docs on how to customize the look and feel. I'm using `@remirror/styles/emotion`. Are there any examples? 

I'm using MUI 5 and basically trying to remove styles from Remirror and wrap things in MUI instead.

So far I've tried

```
    <RemirrorThemeProvider
      theme={{
        color: {
          outline: "white",
        },
        boxShadow: {
          1: "none",
          2: "none",
          3: "none",
        },
      }}
    >
```

this removes the box shadow when the cursor is in the editable area (the blue border is coming from a MUI `<Box>`:

![image](https://user-images.githubusercontent.com/86107/151572252-ab07384f-d61b-45d6-8c71-b71dab10d1bc.png)

but when it's not, the shadow shows up again:

![image](https://user-images.githubusercontent.com/86107/151572324-811ae4df-f779-4af0-84b6-dd4f6a5974c7.png)

Are there different states that need to be styled?

**Comment by pplonski** [comment]
I'm also looking for more examples on how to set custom style, for example how to change `--rmr-color-primary` variable?

**Comment by pplonski** [comment]
OK, I manage to set colors in editor. 

```jsx
import { WysiwygEditor } from "@remirror/react-editors/wysiwyg";

export default function HomeView() {
  return (
    <div>
      <div className="container">
        <div className="row">
          <div className="col-md-10 offset-md-1">
            <h1>Home</h1>
            <div style={{ padding: 16 }}>
              <WysiwygEditor placeholder="Enter text..." theme={{ 
                fontFamily: { default: 'cursive, "Gill Sans", Georgia' },
                color: {
                  foreground: "#e0e",
                  background: "#aaa",
                  primary: "#f00",
                  secondary: "#00f",
                  muted: "#0ee",
                  faded: "#ee0",
                  border: "#eee",
                  shadow1: "#550",
                  shadow2: "#050",
                  shadow3: "#500",
                  outline: "#eee",
                  active: {
                    primary: "#121212",
                    border: "#ee00ee"
                  },
                  hover: {
                    primary: "#121212",
                    border: "#ee00ee"
                  }
                },
                }}/>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
}
```

Thank you for this beautiful package! :+1: :1st_place_medal:
